blocks/card-social-comment/src/CardSocialComment.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { ArrowForward, TurnRight, StarBorder, MoreHoriz } from "@mui/icons-material"
+import {
+  Card,
+  CardContent,
+  Box,
+  Avatar,
+  List,
+  ListItem,
+  Link,
+  ButtonGroup,
+  IconButton,
+  Typography,
+  ColorPaletteProp
+} from "@mui/joy";
+
+export default function CardSocialComment() {
+  const comments = [
+    {
+      commentor: 'Edward Clark',
+      commentAt: '14:07',
+      comments: [
+        <Typography>The biggest change of design in 2019 will be the makeup of the people doing the actual design work.</Typography>,
+        <Typography><Link underline="none" variant="soft" color="primary">@Steve</Link> please read this.</Typography>
+      ],
+      color: 'success',
+    },
+    {
+      commentor: 'Albert Ox',
+      commentAt: '13:37',
+      comments: [
+        <Typography>We absolutely need to get smarter about privacy. Society is creeped out by big tech and big data.</Typography>
+      ],
+      color: 'warning',
+    }
+  ]
+
+  const convertNameToAvatarText = (name: string) => name.split(" ").reduce((acc, word) => `${acc}${word[0] || ''}`, '')
+
+  return (
+    <Card sx={{ borderRadius: "xl" }}>
+      <CardContent>
+        <List sx={{ "--ListItem-paddingLeft": 0, "--ListItem-paddingRight": 0 }}>
+          {
+            comments.map(({ commentor, commentAt, comments, color }) => (
+              <ListItem>
+                <Box display="flex" width="100%">
+                  <Avatar color={color as ColorPaletteProp}>{convertNameToAvatarText(commentor)}</Avatar>
+                  <Box width="100%" marginLeft="8px" marginBottom="8px">
+                    <Box display="flex" justifyContent="space-between" alignItems="flex-end" marginBottom="4px">
```

Comment:
```suggestion
                    <Box display="flex" justifyContent="space-between" alignItems="center" marginBottom="4px">
```

`center` looks better.